lib/lita/extensions/confirmation.rb

```diff
@@ -24,6 +24,7 @@ def call
           message.reply(
             I18n.t(
               "lita.extensions.confirmation.request",
+              prefix: @robot.alias ? @robot.alias : "#{@robot.mention_name} ",
```

Comment:
So, I tried doing ``robot.mention_format(robot.mention_name)``, and ran into:

``` ruby
      SystemExit:
        exit
```

anytime I tried to reference method_format (even with Lita dependency bumped up to 4.4).  I tried digging into how this was exposed from Lita / the adapter, but to no avail so far.  Any pointers?